src/Core/composer.json

```diff
@@ -18,7 +18,7 @@
         "psr/cache": "^1.0",
         "psr/log": "^1.0",
         "symfony/deprecation-contracts": "^2.1",
-        "symfony/http-client": "^4.4 || ^5.0,!=5.2.0",
+        "symfony/http-client": "^4.4.16 || ^5.1.7,!=5.2.0",
```

Comment:
Not really related to this PR, but the PR reveal the bug.
I old versions (prior this PR https://github.com/symfony/symfony/pull/38551) the http-client throws error when the content-type were not occurate.

Which is the case for localstack **AND** for few endpoint in AWS. We once maintained a custom implementation of `toArray`, but when we removed this patch we forgot to bump the composer.json